src/index.js

```diff
@@ -121,6 +121,11 @@ const getAssetFromKV = async (event, options) => {
   if (response) {
     let headers = new Headers(response.headers)
     headers.set('CF-Cache-Status', 'HIT')
+    if(options.browserTTL === null ){
```

Comment:
looks like this needs to go through prettier

Reply:
sorry i changed my vscode config been driving me nuts!